### PR TITLE
Share all api to elioctl

### PR DESCRIPTION
**Pull request by kgermanov**
Share all IOCTL API on elioctl

### CODE REVIEW

**Review by anelson — COMMENTED** [review]
This LGTM, but @e-kov  or @skypodolsky should approve before merging.

**Comment by e-kov** [comment]
I'm not sure what is the purpose of this change, especially about `info` IOCTL. Because new added command `elioctl info X` duplicates output of the `cat /proc/elastio-snap-info`. And you could just read this file from `elioctl` and parse a part by minor, probably )

Anyway, there are few minor remarks:
1. The new commands are missing in the bash completion file
https://github.com/elastio/elastio-snap/blob/cc5dd438e996adf0491d21c123480ccd0a33a0be/app/bash_completion.d/elioctl#L7
2. `get-free-minor` w/o `sudo` returns wrong error: 
`error interpreting get_free_minor parameters: Numerical result out of range`
instead of
`driver returned an error performing specified action. check dmesg for more info: Permission denied`.

**Comment by kgermanov** [comment]
About **info** - yes, we can parse **/proc/elastio-snap-info**, But problems in that we need **parse** json. With this subcommand we can simple do such:
```
root@user-vm:/# elioctl info 1 | grep "fallocate" | cut -d ':' -f 2 |  grep -oP '(\w*)'
7462295961
```
Main goal in that elioctl share **all** available API. As result client does not need write code to link with library.

File: app/elioctl.c [hunk 1]
```diff
@@ -316,6 +320,83 @@ static int handle_reconfigure(int argc, char **argv){
...
+
+	if(argc != 2){
+		errno = EINVAL;
+		goto error;
+	}
+
+	ret = parse_ui(argv[1], &minor);
+	if(ret) goto error;
+
+	ret = elastio_snap_info(minor, &info);
```
[the remark this answers is not in the document]
Comment (e-kov) [reply app/elioctl.c:337]: This is only one thing to fix before the merge.

[the remark this answers is not in the document]
Comment (kgermanov) [reply app/elioctl.c:337]: Same for another's parse_ui:
```
root@user-vm:/# app/elioctl destroy -1
error interpreting destroy parameters: Success

root@user-vm:/# app/elioctl destroy bad_value
error interpreting destroy parameters: Success
```

**Review by e-kov — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -126,7 +126,7 @@ jobs:
             if $(which apt-get >/dev/null 2>&1); then
               export DEBIAN_FRONTEND=noninteractive
               sudo apt-get update
-              sudo -E apt-get install -y lvm2 mdadm
+              sudo -E apt-get install -y --force-yes lvm2 mdadm
```
Comment (skypodolsky) [inline .github/workflows/ci.yml:129]: I'm not sure we need this. Could you please elaborate on that?

**Review by skypodolsky — COMMENTED** [review]
I've just added a single question, but other than that it looks good to me

File: .github/workflows/ci.yml (hunk 2 above)
Comment (skypodolsky) [reply .github/workflows/ci.yml:129]: Ah, I see from logs that it failed without this flag on Debian 8.

**Review by skypodolsky — APPROVED** [review]

**Pull request merged**